app/styles/main.scss

```diff
@@ -23,6 +23,17 @@ html, body {
   height: 100%;
 }
 
+.flexbox-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  min-height: 100vh;
+}
+
+.flexbox-spacer {
+  flex-grow: 2;
```

Comment:
Since the `min-height` of the `.flexbox-container` is 100% of the screen height, the div takes up no space unless the content doesn't fill the screen. If the content does not fill the screen, the `.flexbox-spacer` div will expand, pushing the footer to the bottom of the page, and the body towards the nav.